.drone.yml

```diff
@@ -6,7 +6,6 @@ pipeline:
     image: webhippie/hugo:latest
     commands:
       - apk -U add git
-      - git clone https://github.com/go-gitea/theme.git themes/gitea
```

Comment:
`git submodule update` would not do anything as submodules were just initialized.
I guess you wanted to instead `pull` the submodule to be on the tip of the master branch in its remote ?

It depends on what you want drone to do: use the theme approved in the master branch of `website` or use whatever happens to be in the master branch of `theme`